src/modules/value-editor/viewmodel.cpp

```diff
@@ -251,3 +280,30 @@ void ValueEditor::ViewModel::removeModel(QSharedPointer<ValueEditor::Model> mode
     m_valueModels.removeAt(i);
     endRemoveRows();
 }
+
+void ValueEditor::ViewModel::removeModel(const QString& keyName)
```

Comment:
Update ValueEditor::ViewModel::closeDbKeys::closeDbKeys method instead:
Add arg `QRegExp pattern` with "*"  as default value and match key name in condition like here https://github.com/uglide/RedisDesktopManager/blob/0.8.0/src/modules/connections-tree/items/databaseitem.cpp#L268 
PS: QRegExp should have QRegExp::Wildcard matching type